ignite/metrics/recall.py

```diff
@@ -115,8 +115,14 @@ def update(self, output: Sequence[torch.Tensor]) -> None:
 
         if self._type == "multilabel":
             if not self._average:
-                self._true_positives = torch.cat([self._true_positives, true_positives], dim=0)
-                self._positives = torch.cat([self._positives, actual_positives], dim=0)
+                self._true_positives = torch.cat(
```

Comment:
Here it should be almost the same as for precision (L180-L181), right ?
```python
                self._true_positives = torch.cat([cast(torch.Tensor, self._true_positives), true_positives], dim=0)
                self._positives = torch.cat([cast(torch.Tensor, self._positives), all_positives], dim=0)
```

Reply:
yeah, you are right, but it is so strange. My IDE is totally fine with using `cast`, but `mypy` still complains with
```
error: Cannot determine type of '_true_positives'  [has-type]
error: Cannot determine type of '_positives'  [has-type]
```
I even tried the newer `mypy` version, still same error 😢